asserts/snapasserts/validation_sets.go

```diff
@@ -449,3 +449,68 @@ func (v *ValidationSets) CheckInstalledSnaps(snaps []*InstalledSnap) error {
 	}
 	return nil
 }
+
+func (v *ValidationSets) constraintsForSnap(snapRef naming.SnapRef) *snapContraints {
```

Comment:
Ouch :-) Maybe add another PR (before or after this one) where `snapContraints` is renamed to `snapConstraints`?

Reply:
oops, I agree

Reply:
nice catch and I didn't notice because of auto-completion in vscode ;)

Reply:
I'll address this in a followup.